app/modules/transactions.js

```diff
@@ -72,9 +72,11 @@ export const sendTransaction = (sendAddress: string, sendAmount: string, symbol:
     let sendAssetFn
     if (symbol === ASSETS.NEO || symbol === ASSETS.GAS) {
       sendAssetFn = () => api.neonDB.doSendAsset(net, sendAddress, publicKeyOrWif, sendAsset, asyncSigningFunction)
-    } else {
+    } else if (!isHardwareSend) {
       const scriptHash = getScriptHashForNetwork(net, symbol)
       sendAssetFn = () => api.nep5.doTransferToken(net, scriptHash, publicKeyOrWif, sendAddress, adjustDecimalAmountForTokenTransfer(parsedSendAmount), 0, asyncSigningFunction)
+    } else {
+      return rejectTransaction('ledger support is not yet ready for sending NEP5 tokens')
```

Comment:
Looks good! Maybe capitalize `ledger` (like the previous message)

Reply:
@evgenyboxer will do. also, this should never be called since I disabled the send button, but always good to make sure. we can put this back once ledger support is added